telescope/html/js/app/components/Project.mjs

```diff
@@ -26,12 +39,21 @@ export default class Project extends Component {
       (c) => html`
         <${FocusedCheck.Consumer}>
           ${(focusedCheckContext) => html`
-            <${Check}
-              data="${c.data}"
-              result="${c.result}"
-              fetchCheckResult="${fetchCheckResult}"
-              focusedCheckContext="${focusedCheckContext}"
-            />
+            <${SelectedTags.Consumer}>
+              ${(SelectedTagsContext) => {
+                if (!checkMatchTags(c.data, SelectedTagsContext.tags)) {
+                  return "";
+                }
+                return html`
```

Comment:
This feels like more personal preference than anything, but an else branch or ternary might be nice here.